### PR TITLE
Show pending+exchange balance in trade

### PR DESCRIPTION
**Pull request by Velenir**
If you want that behavior

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr241--dexreact.review.gnosisdev.com

File: src/components/TradeWidget/TokenRow.tsx [hunk 1]
```diff
@@ -210,7 +210,11 @@ const TokenRow: React.FC<Props> = ({
             <strong>
               <Link to="/deposit">Exchange wallet:</Link>
             </strong>{' '}
-            <span className="success">{formatAmount(exchangeBalanceAndPendingBalance, balance.decimals) || '0'}</span>
```
Comment (alfetopito) [inline src/components/TradeWidget/TokenRow.tsx]: `formatAmount` already deals with `null` inputs, this code is redundant.

Comment (Velenir) [reply src/components/TradeWidget/TokenRow.tsx]: That check is for `balance.decimals`. Changed var to be more understandable.

**Comment by alfetopito** [comment]
@Velenir It was merged into `use_latest_contracts`.
Can you also merge this into `develop`?

**Comment by Velenir** [comment]
> @Velenir It was merged into `use_latest_contracts`.
> Can you also merge this into `develop`?

Done

**Pull request merged**